back/app/services/timeline_service.rb

```diff
@@ -11,7 +11,7 @@ def future_phases(project, time = Time.now)
   def past_phases(project, time = Time.now)
     date = time.in_time_zone(AppConfiguration.instance.settings('core', 'timezone')).to_date
     project.phases.select do |phase|
-      phase.end_at < date
+      phase.end_at.present? && phase.end_at < date
```

Comment:
Are there any other places where we need to add such a check (probably, a simple safe navigation operator `&` would be fine in many of them) https://github.com/search?q=repo%3ACitizenLabDotCo%2Fcitizenlab%20phase.end_at&type=code

E.g.
```
back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/project_phase_upcoming.rb:80
back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/project_phase_started.rb:82
back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/admin_digest.rb:244
back/app/models/idea.rb:186
...
```

Reply:
Yeah will check this in more details, but there is another PR I've got open  for email campaigns etc

Reply:
Not sure why I've not used a safe navigation operator here. Will change if there was no reason

Reply:
Sorry for the confusion, I didn't mean that the safe navigation operator here is better than `present?`, I just wanted to ask about other files like `back/app/models/idea.rb:186` where we call `end_at`. But if it's handled in another PR, it's fine.